site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Restore Hooks"
+layout: docs
+---
+
+Velero now supports Restore Hooks, custom actions that can be executed during or after the restore process. There are two kinds of Restore Hooks:
+
+1. InitContainerRestoreHooks: These will add init containers into restored pods to perform any necessary setup before the application containers of the restored pod can start.
+1. ExecRestoreHooks: These can be used to execute custom commands or scripts in containers of a restored Kubernetes pod.
+
+Only InitContainerRestoreHooks is supported and support for ExectRestoreHooks is COMING SOON!
+
+## InitContainer Restore Hooks
+
+As previously mentioned, these hooks can be used to add init containers into pods, before them being restored, to have the init containers perform preparatory actions for the restored pod to resume running from its backed-up state.
```

Comment:
"Use an `InitContainer` hook to add init containers into a pod before it's restored. You can use these init containers to run any setup needed for the pod to resume running from its backed-up state."

Rewrite suggestion to use active voice